Bottom: 0ec26fddefc0cd3b409f01e273810390e2d51b75
Top:    44ac81a2cca3d9ea0546a50286daf2b70b58dd53
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-17 19:33:59 -0400

Refresh of i2s-dma

---

diff --git a/sound/soc/lpc313x/lpc313x-i2s.c b/sound/soc/lpc313x/lpc313x-i2s.c
index e536618..ffd86bbe5 100644
--- a/sound/soc/lpc313x/lpc313x-i2s.c
+++ b/sound/soc/lpc313x/lpc313x-i2s.c
@@ -135,8 +135,6 @@ lpc31xx_i2s_read(const struct lpc31xx_i2s_info *info, uint32_t reg)
 	return value;
 }
 
-
-
 static inline int lpc313x_get_ch_dir(struct snd_pcm_substream *substream)
 {
 	int dir = CH_REC;
